new/backend.js

```diff
@@ -116,7 +116,11 @@ const scripts = mode => {
   let controllerTemplate = loadFile('./files/scripts/backend/templates/controller.js')
   let routesTemplate = loadFile('./files/scripts/backend/templates/routes.js')
 
-  helpers.addScriptToNewPackageJSON('start', 'nodemon server/cluster.js')
+  if (mode === 'standard') {
+    helpers.addScriptToNewPackageJSON('start', `nodemon --watch server server/cluster.js --mode='development'`)
```

Comment:
I need to remove the --mode='development' its not necessary as the env is set within the webpack config file